modules/tinymce/src/plugins/table/main/ts/ui/Buttons.ts

```diff
@@ -165,6 +168,56 @@ const addButtons = (editor: Editor, selectionTargets: SelectionTargets, clipboar
     icon: 'table'
   });
 
+  const tableClassList = getTableClassList(editor);
+
+  if (tableClassList.length === 0) {
+    // eslint-disable-next-line no-console
+    console.error('Missing table class list');
+  } else {
+    editor.ui.registry.addMenuButton('tableclass', {
+      icon: 'table-classes',
```

Comment:
This relies on the icons being merged first, so we'll need that merged before we can merge this PR.

- [ ] Don't merge until https://github.com/tinymce/tinymce/pull/6613 has been merged